xs2a-adapter-service-api/src/main/java/de/adorsys/xs2a/adapter/service/AspspModifyRepository.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018-2018 adorsys GmbH & Co KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.adorsys.xs2a.adapter.service;
+
+import de.adorsys.xs2a.adapter.service.model.Aspsp;
+
+public interface AspspModifyRepository {
```

Comment:
These "modify" infixes look horrible to me. Have you considered any alternatives?
Perhaps `AspspRepository` should be `AspspReadonlyRepository` and `AspspModifyRepository` extend it and be named `AspspRepository`.
Also please use standard method names https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html

Reply:
renamed AspspRepository to AspspReadonlyRepository

Reply:
In my opinion this doesn't cut it.

Reply:
please suggest a better name.